pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -35,8 +35,8 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 %link(rel='stylesheet' type='text/css' href='/shared/css/user-progress.css')
 
 - js_locale = request.locale.to_s.downcase.tr('-', '_')
-%script{src: CDO.studio_url("/blockly/js/#{js_locale}/common_locale.js")}
-%script{src: CDO.studio_url('/blockly/js/teacher-dashboard/index.js')}
+%script{src: asset_path("js/#{js_locale}/common_locale.js")}
```

Comment:
We appear not to generate minified versions of locale files. https://studio.code.org/p/applab for examples serves minified applab.min-[digest].js and unminified applab_locale-[digest].js. I suspect the reason is that locale files consist primarily of strings and keys that can't or shouldn't be minified.